.circleci/config.yml

```diff
@@ -26,6 +26,7 @@ jobs:
 
       - run: echo ${CIRCLE_BRANCH}
 
+      - run: set +e
```

Comment:
Usually I see `set -e` to enable this mode. Why did you decide to disable it?

Reply:
I thought set +e enables failures.  That is the status of the build reflects the last run task.

Reply:
It is a little weird because the API is the opposite of what one might expect. `-` will "set" the option, and `+` will "unset" the option.

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

>Using ‘+’ rather than ‘-’ causes these options to be turned off. The options can also be used upon invocation of the shell. The current set of options may be found in $-.